prowler/providers/aws/services/awslambda/awslambda_function_vpc_is_in_multiple_az/awslambda_function_vpc_is_in_multiple_az.py

```diff
@@ -0,0 +1,47 @@
+from typing import List
+
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.awslambda.awslambda_client import awslambda_client
+from prowler.providers.aws.services.awslambda.awslambda_function_inside_vpc.awslambda_function_inside_vpc import (
+    awslambda_function_inside_vpc,
+)
+from prowler.providers.aws.services.vpc.vpc_client import vpc_client
+
+
+class awslambda_function_vpc_is_in_multiple_az(Check):
+    def execute(self) -> List[Check_Report_AWS]:
+        findings = []
+        LAMBDA_MINIMAL_AZ = awslambda_client.audit_config.get("lambda_minimal_az", 2)
+        for function_arn, function in awslambda_client.functions.items():
+            # only proceed if check "awslambda_function_inside_vpc" did not run or did not FAIL to avoid to report that the function is not inside a VPC twice
+            if not awslambda_client.is_failed_check(
+                awslambda_function_inside_vpc.__name__,
+                function_arn,
+            ):
```

Comment:
I cannot reproduce this logic, please take it a look and add a test.

Reply:
Added the logic of failed check to other check and added one test to ensure everything is working as expected.